src/test/java/uk/gov/pay/api/it/CreateAgreementIT.java

```diff
@@ -66,12 +70,25 @@ public void shouldReturn400WhenWhenReferenceIsNull() {
                 .body("description", is("Missing mandatory attribute: reference"));
     }
 
+    @Test
+    public void shouldReturn400WhenWhenDescriptionIsNull() {
```

Comment:
Tests checking we reject an empty string and something that’s too long would be good.

Reply:
Yeah, I don't really like that these are done at an integration level or that they're done per property but I could increase the coverage here.